content/tutorials/xrp-ledger-businesses/list-your-exchange-on-xrp-charts.md

```diff
@@ -0,0 +1,197 @@
+# List Your Exchange on XRP Charts
+
+In addition to providing data about the XRP Ledger network and its accounts and transactions, XRP Charts also provides [XRP market data](https://xrpcharts.ripple.com/#/xrp-markets) from external exchanges. This tutorial describes how to have your exchange and its XRP trade and order book data listed on XRP Charts.
+
+To enable XRP Charts to list your exchange, complete the following tasks:
+
+1. [Provide a Recently Executed Trades RESTful API endpoint](#provide-a-recently-executed-trades-endpoint).
+
+2. [Provide a Current Order Book RESTful API endpoint](#provide-a-current-order-book-endpoint).
+
+3. [Send an exchange listing request to XRP Charts](#send-an-exchange-listing-request-to-xrp-charts).
+
+Your exchange may have existing RESTful API endpoints that can deliver data in the formats described in this document. If so, there may be little to no development effort required on your part to complete tasks 1 and 2.
+
+If you have any questions about endpoint specifications, contact xxx@xxxxx.com. ***TODO: do we want to provide a way for them to contact us for dev support?***
+
+
+## Provide a Recently Executed Trades Endpoint
+
+Provide a RESTful API endpoint that returns the most recent 500-1,000 individual trades executed in a particular XRP market.
+
+To ensure that it doesn't miss a trade, XRP Charts queries the endpoint frequently, between every 5 and 30 seconds, aiming to get responses that have overlapping trade data. Ensure that any rate limit enforced by your endpoint can accommodate this query frequency. XRP Charts records unique trade data only, even if it gets overlapping trades.
+
+If XRP Charts needs to query your endpoint at a frequency that exceeds your rate limit, XRP Charts may request that you adjust the rate limit or provide the `last_tid` parameter. ***TODO: okay to state this? accurate?***
```

Comment:
This is accurate . Typically we have pre defined rate limiting set in our data infrastructure , this is done by maintaining a record of the IP .